specs/signature-envelope-cose.md

```diff
@@ -193,13 +197,13 @@ The final signature envelope is a `COSE_Sign1_Tagged` object, consisting of Payl
 
 ### Supported `alg` header values
 
-Notary v2 implementation MUST enforce the following constraints on signature generation and verification:
+Notary Project specification implementation MUST enforce the following constraints on signature generation and verification:
```

Comment:
The Notary Project specification implementation MUST

Reply:
I will update it to `Implementations of the Notary Project`